backend/services/email_service.py

```
@@ -0,0 +1,37 @@
+    body = email_data.get("body", "").lower()
+    return "please reply" in body or "?" in body
+
+def process_self_to_self(email_data: Dict[str, Any], user_email: str) -> bool:
+    """
+    Detects if an email is sent from the user to themselves, turning it into a knowledge node.
+    """
+    sender = email_data.get("sender", "")
+    recipients = email_data.get("recipients", "")
+    
+    if user_email in sender and user_email in recipients:
+        logger.info(f"Self-to-self email detected. Organizing as knowledge node.")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)